lib/manticore_jars.rb

```diff
@@ -1,8 +1,18 @@
 # this is a generated file, to avoid over-writing it just delete this comment
-require "jar_dependencies"
+begin
+  require 'jar_dependencies'
+rescue LoadError
+  require 'commons-logging/commons-logging/1.2/commons-logging-1.2.jar'
+  require 'commons-codec/commons-codec/1.15/commons-codec-1.15.jar'
+  require 'org/apache/httpcomponents/httpcore/4.4.14/httpcore-4.4.14.jar'
+  require 'org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar'
+  require 'org/apache/httpcomponents/httpmime/4.5.13/httpmime-4.5.13.jar'
+end
 
-require_jar("commons-logging", "commons-logging", "1.2")
-require_jar("org.apache.httpcomponents", "httpmime", "4.5.2")
-require_jar("commons-codec", "commons-codec", "1.10")
-require_jar("org.apache.httpcomponents", "httpclient", "4.5.2")
-require_jar("org.apache.httpcomponents", "httpcore", "4.4.4")
+if defined? Jars
```

Comment:
Why the two different ways of loading jars now?

Reply:
this file is autogenerated, the change is likely due newer jar_dependencies